### DISCUSSION TITLE
<Content> tag in NativeBase 3?

### DISCUSSION THREAD

**Discussion by aaa646123** [GeekyAnts/NativeBase#5379] [Feature request]
### Description

I am trying upgrade the module from v2 to v3.
But I found that seems <Content> tag didn't exist in v3.
Does it change the name?

### Problem Statement

I want to upgrade module from v2 into v3.

### Proposed Solution or API

```
import { Container, Content, Text } from 'native-base';

const LoadingScreen = ({ text }) => (
  <Container>
    <Content contentContainerStyle={commonStyle.centerRowContent}>
      <Text>{text}</Text>
    </Content>
  </Container>
);
```

**Comment by ankit-tailor** [comment]
Hey @aaa646123 , `Content` component is deprecated in v3. We'll add it to our roadmap for future releases.
